rusoto/core/Cargo.toml

```diff
@@ -63,6 +64,7 @@ serde_test = "1.0.1"
 
 [features]
 default = ["native-tls"]
+encoding = []
```

Comment:
To achieve the behavior in the comment above, this can be changed to `encoding = ["flate2"]` to pull in the crate as needed. 👍